src/tests/countingCards.test.js

```diff
@@ -1,28 +1,9 @@
-// let count = require("../challenges/countingCards.js")
-const cc = require("../challenges/countingCards.js")
-const cardGame = require("../challenges/countingCards.js")
+const myFunctions = require("../challenges/countingCards.js")
```

Comment:
Can this be expanded out into what is actually exported and imported here rather than importing into an object like thing.